src/pages/Login.vue

```diff
@@ -395,16 +397,32 @@ export default {
       // whether the phone number entered by the user is valid
       return this.phoneInput.toString().match(/^([0]|\+91)?[6-9]\d{9}$/g) != null;
     },
+    startCountdown(seconds) {

```

Comment:
better name + since you are going to use 60 only, you can use it as the default value and so, you wouldn't have to pass `60` while calling the function.

```suggestion
    startResendOTPTimer(seconds = 60) {
```